access/src/main/external/static/recordTransformations/modsToFullRecord.xsl

```diff
@@ -774,14 +781,66 @@
 			<tr>
 				<xsl:variable name="groupKey" select="current-grouping-key()"/>
 				<th>
-					<xsl:choose>
-							<xsl:when test="$groupKey = 'isReferencedBy'">
-								<xsl:value-of>Referenced By</xsl:value-of>
-							</xsl:when>
-							<xsl:otherwise>
-								<xsl:value-of select="concat(upper-case(substring($groupKey,1,1)), substring($groupKey,2))"/>
-							</xsl:otherwise>
-						</xsl:choose>
+				    <xsl:choose>
```

Comment:
I know I'm a hypocrite for asking this, since it looks like the indentation in the original XSLT created by me was off, but could you adjust the indentation in this xsl:choose block so all the xsl:when's and xsl:otherwise blocks are indented once from the xsl:choose?  Without indentation this world would fall into chaos